modules/rosa-cluster-autoscaler-ui-settings.adoc

```diff
@@ -16,25 +18,45 @@ The tables explain all the configurable UI settings when using cluster autoscali
 |Type or Range
 |Default
 
-|`balanceSimilarNodeGroups`
+|`balance-similar-node-groups`
```

Comment:
Apologies for that, but now the order of the fields is slightly different in the UI. It was done to keep together `balance-similar-node-groups` and `balancing-ignored-labels`.

I just sent you an updated screenshot.

Reply:
Thanks, I'll make the changes. :+1: